package.json

```diff
@@ -59,13 +59,13 @@
     "@sentry-internal/rrweb-player": "2.25.0",
     "@sentry-internal/rrweb-snapshot": "2.25.0",
     "@sentry/babel-plugin-component-annotate": "^2.16.1",
-    "@sentry/core": "^8.18.0",
-    "@sentry/node": "^8.18.0",
-    "@sentry/react": "^8.18.0",
+    "@sentry/core": "^8.20.0",
```

Comment:
Needed for most up-to-date behavior of the SDK functions (older version throws, newer logs to debug)

Reply:
What's different in the new version? We should be building in a way that older SDK versions have a chance to work too, and if not then we should build-in checks and messages so the app doesn't throw when users access it!

Reply:
The new version is more "lenient" with errors. For example getReplayId used to throw if no recording session is active - now it returns undefined. `start()` and `stop()` would also throw if called in the wrong state. And `flush()` would start a replay if in session mode.

I'll double check all this with a diff on the SDK code

Reply:
Left a comment in the code. The changes were made in 8.20: https://github.com/getsentry/sentry-javascript/pull/13000

Reply:
This PR will work without bumping the SDK.

Reply:
It's generally a good idea to keep them as separate PRs